snuba/settings/settings_test_distributed.py

```diff
@@ -0,0 +1,39 @@
+import os
+from typing import Set
+
+TESTING = True
+
+REDIS_DB = 2
+STATS_IN_RESPONSE = True
+CONFIG_MEMOIZE_TIMEOUT = 0
+
+RECORD_QUERIES = True
+USE_RESULT_CACHE = True
+
+SENTRY_DSN = os.getenv("SENTRY_DSN")
+
+SKIPPED_MIGRATION_GROUPS: Set[str] = {"metrics"}
+
+CLUSTERS = [
+    {
+        "host": os.environ.get("CLICKHOUSE_HOST", "localhost"),
+        "port": int(os.environ.get("CLICKHOUSE_PORT", 9000)),
+        "user": os.environ.get("CLICKHOUSE_USER", "default"),
+        "password": os.environ.get("CLICKHOUSE_PASSWORD", ""),
+        "database": os.environ.get("CLICKHOUSE_DATABASE", "default"),
+        "http_port": int(os.environ.get("CLICKHOUSE_HTTP_PORT", 8123)),
+        "storage_sets": {
```

Comment:
Is it possible to populate this list automatically? Like by iterating `StorageSetKey`? I am worried about having to remember to update this file whenever we change the storage sets.

Reply:
It is intended. The settings file should not depend on anything else so that everything can depend on it.

https://github.com/getsentry/snuba/blob/master/snuba/clusters/cluster.py validates the integrity of what is written here.

Also there is a requirement to allow invalid storage sets here for deploy safety
https://github.com/getsentry/snuba/commit/c9a3b2ca5a72e47074c89b11ac30562e4e95f019

Reply:
This file is essentially all of the settings from `settings_test` combined with all of the settings from `settings_distributed`. I think we might be able to avoid all of this duplication with something like:
```
from snuba.settings.settings_test import *
from snuba.settings.settings_distributed import *
```

Reply:
Good point; I'll make `settings_test_distributed` more concise.